hotham/src/rendering/camera.rs

```diff
@@ -6,41 +6,41 @@ use crate::util::posef_to_isometry;
 #[derive(Debug, Clone)]
 /// The Camera, or View, in a scene.
 pub struct Camera {
-    /// The camera's position in space
-    pub position: Isometry3<f32>,
+    /// The camera's position in globally oriented stage space
+    pub gos_from_view: Isometry3<f32>,
     /// The view matrix
-    pub view_matrix: Matrix4<f32>,
+    pub view_from_gos: Matrix4<f32>,
 }
 
 impl Default for Camera {
     fn default() -> Self {
         let t = Translation3::from(Vector3::zeros());
         Self {
-            position: Isometry3::from_parts(t, UnitQuaternion::identity()),
-            view_matrix: Matrix4::identity(),
+            gos_from_view: Isometry3::from_parts(t, UnitQuaternion::identity()),
+            view_from_gos: Matrix4::identity(),
         }
     }
 }
 
 impl Camera {
-    /// Update the camera's position from an OpenXR view
-    pub fn update(&mut self, view: &View) -> Matrix4<f32> {
+    /// Update the camera's position from an OpenXR view (stage from view)
+    pub fn update(&mut self, view: &View, gos_from_stage: &Isometry3<f32>) -> Matrix4<f32> {
         // Convert values from OpenXR format
-        let camera_position = posef_to_isometry(view.pose);
-        self.position = camera_position;
+        let stage_from_view = posef_to_isometry(view.pose);
+        self.gos_from_view = gos_from_stage * stage_from_view;
 
-        self.view_matrix = self.build_matrix();
-        self.view_matrix
+        self.view_from_gos = self.build_matrix();
+        self.view_from_gos
     }
 
     /// Get the camera's position
     pub fn position(&self) -> Vector4<f32> {
```

Comment:
I think that this function should be renamed to something like `gos_position` to keep it consistent with the rest of the engine. All other position related functions and variables in Hotham return global space positions